### ISSUE TITLE
Can it be used in react-native

### ISSUE THREAD

**Issue by ndzy-goole** [react-component/field-form#234]

**Comment by preflower** [comment]
yeah, i have use it in react-native; and rc-field-form clearly indicate support for react-native.

**Comment by diyudediyu** [comment]
When using React-Native, add the **component** attribute to **false**.

The following is an example:
import Form from 'rc-field-form';
<Form onFinish={(values) => { console.log("Finish:", values); }} **component={false}**>
